worlds/ufo50/__init__.py

```diff
@@ -51,6 +53,19 @@ class UFO50Web(WebWorld):
     tutorials = [setup_en]
 
 
+class UFO50Item(Item):
```

Comment:
I think this class is going unused.
Everything is dealing with Item and Location instead of UFO50Item and UFO50Location, is that right?